workitem/workitem_repository_blackbox_test.go

```diff
@@ -62,7 +62,7 @@ func (s *workItemRepoBlackBoxTest) TestFailDeleteZeroID() {
 		map[string]interface{}{
 			workitem.SystemTitle: "Title",
 			workitem.SystemState: workitem.SystemStateNew,
-		}, "xx")
+		}, uuid.NewV4())
```

Comment:
ok, I see. Let me create a single identity for tests that create multiple work items, then.